src/pykeen/models/unimodal/complex.py

```diff
@@ -159,7 +159,7 @@ def forward(
         # Compute scores
         return self.interaction_function(h=h, r=r, t=t)
 
-    def score_hrt(self, hrt_batch: torch.LongTensor) -> torch.FloatTensor:  # noqa: D102
+    def score_hrt(self, hrt_batch: torch.LongTensor, use_inverse: bool = False) -> torch.FloatTensor:  # noqa: D102
```

Comment:
what's going on here?

Reply:
Fixed in 1fdd03f45cc24629e33ffcc6f8ca8634dc5ce2e7.